imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -239,6 +240,16 @@ func Test_fdFdstatGet(t *testing.T) {
 	fsc := mod.(*wasm.ModuleInstance).Sys.FS()
 	preopen := fsc.RootFS()
 
+	// replace stdin with a fake TTY file.
+	// TODO: Make this easier once we have in-memory platform.File
```

Comment:
This maybe by next week depending on how much zombification happens during intercontinental back to back travel